sdk/inc/azure/iot/az_iot_common.h

```diff
@@ -60,12 +71,25 @@ enum az_log_classification_iot
 
   AZ_LOG_IOT_ADU
   = _az_LOG_MAKE_CLASSIFICATION(_az_FACILITY_IOT, 4), ///< Azure IoT classification for ADU APIs.
+
+  AZ_LOG_HFSM_IOT_START
+  = _az_LOG_MAKE_CLASSIFICATION(_az_FACILITY_IOT, 5), ///< Azure IoT start event.
+
+  AZ_LOG_HFSM_IOT_ERROR
+  = _az_LOG_MAKE_CLASSIFICATION(_az_FACILITY_IOT, 6), ///< Azure IoT error event.
+
+  AZ_LOG_HFSM_IOT_PROVISIONING_DONE
+  = _az_LOG_MAKE_CLASSIFICATION(_az_FACILITY_IOT, 7), ///< Azure IoT provisioning done event.
 };
 
 enum
 {
   AZ_IOT_DEFAULT_MQTT_CONNECT_PORT = 8883,
-  AZ_IOT_DEFAULT_MQTT_CONNECT_KEEPALIVE_SECONDS = 240
+  AZ_IOT_DEFAULT_MQTT_CONNECT_KEEPALIVE_SECONDS = 240,
+  AZ_IOT_DEFAULT_MIN_RETRY_DELAY_MSEC = 1000,
+  AZ_IOT_DEFAULT_MAX_RETRY_DELAY_MSEC = 100000,
+  AZ_IOT_DEFAULT_MAX_RETRY_JITTER_MSEC = 5000,
+  AZ_IOT_DEFAULT_MAX_HUB_RETRY = 10,
```

Comment:
We can remove the last one for now. The generic ones are useful for the connection management HFSM..